config/initializers/core.rb

```diff
@@ -30,6 +30,14 @@
 Core::Registry::Repository[:cms_api] =
   Core::Repository::CMS::CmsApi.new
 
+if Feature.active?(:cms_footer)
+  Core::Registry::Repository[:footer] =
+    Core::Repository::Footer::CMS.new
+else
+  Core::Registry::Repository[:footer] =
+    Core::Repository::Footer::Static.new
+end
+
```

Comment:
Not cached. Is the extra request a problem?

I'm not at all sure how important this feature is (being able to edit the footer); the only real solution is to drop it if we are worried about the extra request; there isn't enough time to take another approach.